server/src/main/scala/ch.epfl.scala.index.server/DataRepository.scala

```diff
@@ -34,6 +35,8 @@ class DataRepository(github: Github, paths: DataPaths)(private implicit val ec:
         fieldSort("github.forks") missing "0" order SortOrder.DESC mode MultiMode.Avg
       case Some("dependentCount") =>
         fieldSort("dependentCount") missing "0" order SortOrder.DESC mode MultiMode.Avg
+      case Some("contributors") =>
+        scriptSort(script("contributors_size") scriptType ScriptType.FILE) typed "number" order SortOrder.DESC
```

Comment:
Actually it would better to precompute this value. It's less trouble to querry elasticsearch after.

for example: https://github.com/scalacenter/scaladex/blob/master/data/src/main/scala/ch.epfl.scala.index.data/project/ProjectConvert.scala#L299